### DISCUSSION TITLE
Error in MD simulation with Lammps：Non-numeric atom coords - simulation unstable

### DISCUSSION THREAD

**Discussion by liudfgoo** [deepmodeling/deepmd-kit#1180] [Q&A]
I do the AIMD simulation on a bulk liquid water system of 64 water molecules with CP2K. The simulation time is 10 ps and finally I get 10000 frames. Next, I use these data to train the deep potential model. And the training process works well. Both the RMSE of training set and validation set decay to a very small value. Then based on the model, I go on the MD simulation with lammps. However, during the MD simulation, there is a running error: 
**ERROR on proc 0: Non-numeric atom coords - simulation unstable**

I check the dumped .xyz file and visualize it using VMD.  I find the O atoms gradually close and even coincide during the simulation, which leads to the problem. The figure below shows the problem.
![image](https://user-images.githubusercontent.com/49781630/135710974-e42a29f1-7dfd-4543-9096-765172f5513c.png)

I think there are at least two causes. The first is that the training data is limited, in which the frames are all valid and don't include conformations with high energy. The second is that the initial velocity setting is in atomic way in the Lammps MD simulation. At the beginning, the system unit is molecule not atom. And the velocity should be specified in molecular units, which means the H atom and O atom in one molecule should have the same speed.

I want to ask the developers and other users how to solve the problem?
